rest/data/repotracker.go

```diff
@@ -75,7 +76,7 @@ func (c *RepoTrackerConnector) TriggerRepotracker(q amboy.Queue, msgID string, e
 			"repo":    *event.Repo.Name,
 			"ref":     *event.Ref,
 			"branch":  branch,
-			"message": "error occurred while trying to match push event to project refs",
+			"message": "no matching project refs found",
 		}))
 		return err
 	}
```

Comment:
Also this change is correct, since if refs has a zero length the rest of the code in this function is a noop, there is a risk that someone adds something later in this function, not realizing that refs can have a zero length even if `err` is nil. Could you add a check here that exits early?

Reply:
Done